pkg/serial/handshake_test.go

```diff
@@ -129,14 +129,14 @@ func TestHandshakeServerNormalCaseScenario(t *testing.T) {
 
 func TestHandshakeServerLotsOfTrashOnTheLine(t *testing.T) {
 	log.SetLevel(log.InfoLevel)
-	clientConn, serverConn := NewFakeConnection(time.Second * 3)
+	clientConn, serverConn := NewFakeConnection(time.Second * 10)
```

Comment:
If we're playing with test timeouts like this, I think we should be using infinite timeout and letting the Go test timeout kill it.